autoload/handlers.vim

```diff
@@ -302,7 +306,12 @@ def s:processHoverReply(lspserver: dict<any>, req: dict<any>, reply: dict<any>):
     ErrMsg('Error: Unsupported hover contents (' .. reply.result.contents .. ')')
     return
   endif
-  hoverText->popup_atcursor({moved: 'word'})
+  hoverWinid = hoverText->popup_atcursor({moved: 'word'})
+  if hoverWinid > 0
+    call setbufvar(winbufnr(hoverWinid), 'lsphoverpopupid', hoverWinid)
```

Comment:
There is no need to store the window ID as a buffer variable as the user can always get it using the bufwinid() function.

Reply:
perhaps yes, but looks that would not easy, i tried to 'au filetype lsphover' to get it (or whatever user would try to get by this way, i guess would), the bufwinid(bufnr()) is not same value like b:lsphoverpopupid.
anyway, by 'b:lsphoverpopupid' looks would be clear and direct...